ui/src/onboarding/components/AdminStep.tsx

```diff
@@ -204,6 +200,14 @@ class AdminStep extends PureComponent<Props, State> {
     )
   }
 
+  private get passwordError(): string {
+    return this.state.isPassTooShort && 'Password must be at least 8 characters'
```

Comment:
i'm  confused by the intent of this code and what it actually is meant to return. this returns `false` or an error message, but it says it returns a string. I tested this in the console and I got this:

```js
const errorHandler = (arbitraryBool) => { return arbitraryBool && "error message" }

errorHandler(false) // false
errorHandler(true) //  error message
```

am I missing something? Is there a strong preference for doing this on one line instead of doing something like?

```ts
if (this.state.isPassTooShort) {
  return 'Password must be at least 8 characters'
}

return ''
```